content/docs/1.5.0/snapshots-and-backups/backup-and-restore/set-backup-target.md

```diff
@@ -295,3 +296,49 @@ Then, navigate to Longhorn UI > Setting > General > Backup
 You can find an example CIFS backupstore for testing purpose [here](https://github.com/longhorn/longhorn/blob/v{{< current-version >}}/deploy/backupstores/cifs-backupstore.yaml).
 
 **Result:** Longhorn can store backups in CIFS. To create a backup, see [this section.](../create-a-backup)
+
+### Azure Blob Storage Backupstore
+
+Before configuring a Azure Blob Storage backupstore, a credential secret for the backupstore can be created and deployed by
+
+  ```shell
+  #!/bin/bash
+
+  ACCOUNT_NAME: ${Account name of Azure Blob Storage}
+  ACCOUNT_KEY: ${Account key of Azure Blob Storage}
```

Comment:
```suggestion
  # ACCOUNT_NAME: Account Name of Azure Blob Storage
  # ACCOUNT_KEY: Account Key of Azure Blob Storage
```

Reply:
Modified, and added `AZURE_ENDPOINT`. Thanks